Mathlib/Order/SupClosed.lean

```diff
@@ -123,22 +160,69 @@ end SemilatticeInf
 open Finset OrderDual
 
 section Lattice
-variable [Lattice α]
+variable {ι : Sort*} [Lattice α] [Lattice β] {S : Set (Set α)} {f : ι → Set α} {s t : Set α} {a : α}
+
+open Set
+
+/-- A set `s` is *lattice-closed* if `a ⊔ b ∈ s` and `a ⊓ b ∈ s` for all `a ∈ s`, `b ∈ s`. -/
```

Comment:
Is this also the name used in the literature? Or does this concept not exist in the mainstream literature?

Reply:
This concept definitely exists in the literature: it's caled a sublattice. However I am trying to reserve this name for the bundled object, which I'm defining in #7549.

What do you think? Should I also say this is the property of being a sublattice? in the docstring or in the name?

Reply:
I would definitely mention this in the docstring. And I would even consider calling this `isSublattice` or something like that.

Reply:
I guess the argument for the current name is consistency with `SupClosed` and `InfClosed`.

Reply:
I've renamed it to `IsSublattice` for now.